Makefile

```diff
@@ -0,0 +1,90 @@
+	$(eval API_APP_NAME=$(shell jq -r '.api_app_name' terraform/workspace_variables/$(DEPLOY_ENV).tfvars.json))
+
+set-azure-account: ${environment}
+	echo "Logging on to ${AZURE_SUBSCRIPTION}"
+	az account set -s ${AZURE_SUBSCRIPTION}
+
+ci:	## Run in automation environment
+	$(eval DISABLE_PASSCODE=true)
+	$(eval AUTO_APPROVE=-auto-approve)
+	$(eval SP_AUTH=true)
+
+.PHONY: install-fetch-config
+install-fetch-config: ## Install the fetch-config script, for viewing/editing secrets in Azure Key Vault
+	[ ! -f bin/fetch_config.rb ] \
+		&& curl -s https://raw.githubusercontent.com/DFE-Digital/bat-platform-building-blocks/master/scripts/fetch_config/fetch_config.rb -o bin/fetch_config.rb \
+		&& chmod +x bin/fetch_config.rb \
+		|| true
+
+edit-infra-secrets: read-keyvault-config install-fetch-config set-azure-account
+	bin/fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} \
+		-e -d azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} -f yaml -c
+
+print-infra-secrets: read-keyvault-config install-fetch-config set-azure-account
+	bin/fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} -f yaml
+
+validate-infra-secrets: read-keyvault-config install-fetch-config set-azure-account
+	bin/fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} -d quiet \
+		&& echo Data in ${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} looks valid
+
+.PHONY: set-space-developer
+set-space-developer: read-deployment-config ## make dev set-space-developer USER_ID=first.last@digital.education.gov.uk
+	$(if $(USER_ID), , $(error Missing environment variable "USER_ID", USER_ID required for this command to run))
+	cf set-space-role ${USER_ID} dfe ${SPACE} SpaceDeveloper
+
+.PHONY: unset-space-developer
+unset-space-developer: read-deployment-config ## make dev unset-space-developer USER_ID=first.last@digital.education.gov.uk
+	$(if $(USER_ID), , $(error Missing environment variable "USER_ID", USER_ID required for this command to run))
+	cf unset-space-role ${USER_ID} dfe ${SPACE} SpaceDeveloper
+
+stop-app: read-deployment-config ## Stops api app, make dev stop-app CONFIRM_STOP=1
+	$(if $(CONFIRM_STOP), , $(error stop-app can only run with CONFIRM_STOP))
+	cf target -s ${SPACE}
+	cf stop ${API_APP_NAME}
+
+get-postgres-instance-guid: read-deployment-config ## Gets the postgres service instance's guid
+	cf target -s ${SPACE} > /dev/null
+	cf service ${POSTGRES_DATABASE_NAME} --guid
+	$(eval DB_INSTANCE_GUID=$(shell cf service ${POSTGRES_DATABASE_NAME} --guid))
+
+rename-postgres-service: read-deployment-config ## make dev rename-postgres-service NEW_NAME_SUFFIX=old CONFIRM_RENAME
+	$(if $(CONFIRM_RENAME), , $(error can only run with CONFIRM_RENAME))
+	$(if $(NEW_NAME_SUFFIX), , $(error NEW_NAME_SUFFIX is required))
+	cf target -s ${SPACE} > /dev/null
+	cf rename-service  ${POSTGRES_DATABASE_NAME} ${POSTGRES_DATABASE_NAME}-$(NEW_NAME_SUFFIX)
```

Comment:
This won't work as missing variable definition in read-deployment-config. https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/50/files#r882548434